esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -392,6 +413,42 @@ define([
             return retVal;
         },
 
+        _onFileTypeCheckboxClick: function (event) {
```

Comment:
@GordonSmith Looking at the 160 to compare, I saw that the FileType dropdown set as "Not in Superfiles" & the Index checkbox being checked appeared to be a valid combo. That's why this event listener doesn't use the Index checkbox to toggle the "Not in Superfiles" checkbox disabled status.